Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteGPUProxy.h

```diff
@@ -96,7 +96,7 @@ class RemoteGPUProxy final : public PAL::WebGPU::GPU, private IPC::Connection::C
     }
     IPC::Connection& connection() const { return m_gpuProcessConnection->connection(); }
 
-    void requestAdapter(const PAL::WebGPU::RequestAdapterOptions&, CompletionHandler<void(RefPtr<PAL::WebGPU::Adapter>&&)>&&) final;
+    void requestAdapter(const PAL::WebGPU::RequestAdapterOptions&, CompletionHandler<void(RefPtr<PAL::WebGPU::Adapter>&&)>&&, unsigned = 0) final;
```

Comment:
What is the meaning of this? It needs a name. (And it seems unused in the .cpp file)

Reply:
I fixed this up